_layouts/post.html

```diff
@@ -108,7 +108,15 @@
 
   {% if page.discontinuedColumn %}
   <td class="{% unless r.discontinued %}bg-green-000{% endunless %}">
-    {% if r.discontinued %} {{r.discontinued | timeago}} <div>({{r.discontinued | date_to_string}}){% else %}In Production{% endif %}</div>
+    {% if r.discontinued %}
+      {% if r.discontinued == true%}
+        Yes
```

Comment:
Unsure about using "Yes" here. Should we use "Not in Production" instead?

Reply:
How about "Discontinued"?